dotnet/samples/Concepts/Plugins/CreatePluginFromOpenApiSpec_Github.cs

```diff
@@ -97,7 +97,7 @@ public async Task RunOpenAIPluginWithMetadataAsync()
         }
     }
 
-    private static void WriteStringToStream(Stream stream, string input)
+    private static void WriteStringToStream(MemoryStream stream, string input)
```

Comment:
Is this needed ? 
```suggestion
    private static void WriteStringToStream(Stream stream, string input)
```

Reply:
If we don't, we'll need to suppress the analyzer that recommends using the most derived type that works.